src/plugins/vis_type_vega/public/data_model/vega_parser.ts

```diff
@@ -21,7 +21,9 @@ import _ from 'lodash';
 import schemaParser from 'vega-schema-url-parser';
 import versionCompare from 'compare-versions';
 import hjson from 'hjson';
-import { VISUALIZATION_COLORS } from '@elastic/eui';
+// @ts-ignore
+import { euiPaletteColorBlind } from '@elastic/eui/lib/services';
```

Comment:
This should be importable from `@elastic/eui` directly (and then also have typings).